python/ray/serve/_private/http_proxy.py

```diff
@@ -668,22 +759,40 @@ async def send_request_to_replica_streaming(
         )
 
         status_code = ""
+        start = time.time()
+        timeout_s = RAY_SERVE_REQUEST_PROCESSING_TIMEOUT_S
```

Comment:
We should probably use `self.request_timeout_s` for this so the config file can take precedent, but let me know if it's intentional.

Reply:
oh! totally, let me fix that.

Reply:
done